dev/ci/README.md

```diff
@@ -0,0 +1,118 @@
+Continuous Integration
+======================
+
+
+Information for external library / plugin authors
+----------------------------------------------------
+
+Changes to Coq are systematically tested for regression and compatibility
+breakage on our Continuous Integration (CI) platforms *before* integration.
+These tests include the compilation of several external libraries / plugins.
+
```

Comment:
Maybe the intro could be moved before the first section. You could outline the global goals of the CI system, then mention that the document contains information for both "external" developers and "internal" ones.